src/backend/db/videos.js

```diff
@@ -5,17 +5,139 @@
 
 export const videos = [
   {
-    _id: "Wo5dMEP_BbI",
-    title: "Awesome Video about Coding",
+    _id: "uEqYzdz3Zvg",
```

Comment:
Since you have `uuid` package installed, you can let the id get handled by that package.